### ISSUE TITLE
Page template rename suggestion

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/themedd#70]
This might be the OCD in me, but I can't help but feel the "No sidebar" page should be renamed to "Fullwidth" since we have a "Slim" template as well.

**Comment by amdrew** [comment]
There was a reason why I changed it (had it previously called full-width) but I can't recall it now, not even sure if the reasoning is still valid. I'll check my notes!

**Comment by amdrew** [comment]
I believe I originally called it `no sidebar` because the content width of a normal page didn't span the full-width (at the time) - so I refused to call the template full-width because that simply wasn't the case. But then I did make the content span the entire width and introduced the slim template.

I think I'll go ahead and rename the template from no sidebar to full width because it makes more sense, and if there are any issues I'll post back.
